common/taskana-common/src/main/java/pro/taskana/common/internal/util/ObjectAttributeChangeDetector.java

```diff
@@ -40,7 +40,8 @@ public static <T> String determineChangesInAttributes(T oldObject, T newObject)
 
     // this has to be checked after we deal with List data types, because
     // we want to allow different implementations of the List interface to work as well.
-    if (!oldObject.getClass().equals(newObject.getClass())) {
+    if (!oldObject.getClass().equals(newObject.getClass())
+        && !oldObject.getClass().isAssignableFrom(newObject.getClass())) {
```

Comment:
Why is this part of the PR? Didn't we discuss this change in a different context?

Reply:
It is in #2373 but it's not yet integrated yet

Reply:
Do you need this change as part of the PR?